src/analysis/retail/monk/mistweaver/modules/spells/RisingSunKick.tsx

```diff
@@ -46,6 +52,45 @@ class RisingSunKick extends Analyzer {
     this.lastBOK = event.timestamp;
   }
 
+  /** Guide subsection describing the proper usage of RSK */
+  get guideSubsection(): JSX.Element {
+    const explanation = (
+      <p>
+        <b>
+          <SpellLink id={TALENTS_MONK.RISING_SUN_KICK_TALENT.id} />
+        </b>{' '}
+        is one of your primary damaging spells and also an important healing spell due to{' '}
```

Comment:
'is one of your primary damaging spells but is also the highest priority healing spell along side renewing mist' 

-- want to stress the importance of this and ReM as much possible simply because 90% of logs fail these two metrics more than anything else